app/models/course/assessment/answer/scribing_scribble.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 class Course::Assessment::Answer::ScribingScribble < ApplicationRecord
+  validates :creator, presence: true
+
```

Comment:
Need to validate the presence of `answer`?

Reply:
Hmm, schema_validations didn't generate the validation. `answer_id` column is actually nullable in `db/schema.rb`.

Reply:
I can't think of when the scribble should exist without the answer though. Should we fix the schema to be non nullable?